src/open_company_web/components/new_report_popover.cljs

```diff
@@ -0,0 +1,75 @@
+(ns open-company-web.components.new-report-popover
+  (:require [om.core :as om :include-macros true]
+            [om-tools.core :as om-core :refer-macros [defcomponent]]
+            [om-tools.dom :as dom :include-macros true]
+            [om-bootstrap.random :as r]
+            [om-bootstrap.button :as b]))
+
+(def current-year (.getFullYear (new js/Date)))
+
+(def years (range 1950 (+ current-year 1)))
+(def periods ["M1"
```

Comment:
elsewhere we have the periods as a map to their english expansion. I wonder if we can't just use the keys from that other data structure (keys other-thing) to DRY this up.